base/deprecated.jl

```diff
@@ -28,8 +28,7 @@ macro deprecate(old, new, ex=true)
             ex ? Expr(:export, esc(old)) : nothing,
             :(function $(esc(old))(args...)
                   $meta
-                  depwarn(string($oldname, " is deprecated, use ", $newname, " instead."),
-                          $oldmtname)
+                  depwarn($"$old is deprecated, use $new instead.", $oldmtname)
```

Comment:
If you ise `old` and `new` directly, you can remove the `oldname` and `newname` expressions above, right?